ampersand-form-view.js

```diff
@@ -50,6 +52,14 @@ module.exports = View.extend({
         }
 
         if (opts.values) this._startingValues = opts.values;
+        
+        if (this.validCallback) {
+            this.on('change:valid', function(view, validBool) {
+                this.validCallback(validBool);
```

Comment:
The current behaviour is that `validCallback` is called when `valid` changes between true and false. I believe this has been the case since v1.0.0.

In the docs:

```js
validCallback: function (valid) {
    if (valid) {
        console.log('The form is valid!');
    } else {
        console.log('The form is not valid!');
    }
}
```

To me this behaviour make sense.